tools/src/main/java/org/apache/rocketmq/tools/command/acl/GetAccessConfigSubCommand.java

```diff
@@ -52,20 +52,19 @@ public class GetAccessConfigSubCommand implements SubCommand {
     @Override public Options buildCommandlineOptions(Options options) {
         OptionGroup optionGroup = new OptionGroup();
 
-        Option opt = new Option("b", "brokerAddr", true, "query acl config version for which broker");
+        Option opt = new Option("b", "brokerAddr", true, "query acl config for which broker");
```

Comment:
Pls help us change the description to "query acl config for specified broker" to make it more formal and consistent with the clusterName one.